dev/NavigationView/NavigationView.xaml

```diff
@@ -323,7 +323,7 @@
 
                                     <local:NavigationViewItem
                                             x:Name="SettingsTopNavPaneItem"
-                                            Style="{ThemeResource NavigationViewSettingsItemStyleWhenOnTopPane}"
```

Comment:
Do we need to make a pass over all the control templates to ensure we don't have the same issue elsewhere?

Reply:
I went through them and didn't see any control may have this problem yet.
Reveal/Acrylic may have problem but I never see a repro.